### PR TITLE
Issue 905 blower sub assembly

### PR DESCRIPTION
**Pull request by RakurthiLokesh**
The files required for the Blower assembly are added and the assembly procedure is updated to allow only filters air to enter the blower.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -11,4 +11,4 @@
 *.png filter=lfs diff=lfs merge=lfs -text
 *.adicht filter=lfs diff=lfs merge=lfs -text
 *.stl filter=lfs diff=lfs merge=lfs -text
-*.dxf filter=lfs diff=lfs merge=lfs -text
+*.SLDPRT filter=lfs diff=lfs merge=lfs -text
```
Comment (martukas) [inline .gitattributes:14]: What is SLDPRT?

File: manufacturing/blower/README.md [hunk 2]
```diff
@@ -92,3 +92,20 @@ You will have to crimp connectors to the second end of the wiring that connects
 to the [PCB](../../pcb).
 It depends on your build layout, but in the case that you need longer wires, you will have to crimp both ends.
 In such a case you might need a greater number of XH and VH connector components listed in the BOM above.
+
+## Assembly Instructions of Blower to the Enclosure
+
```
Comment (martukas) [inline manufacturing/blower/README.md:97]: I do not think this is the correct sequence for assembling. It must start with the driver.

File: manufacturing/blower/README.md [hunk 3]
```diff
@@ -92,3 +92,20 @@ You will have to crimp connectors to the second end of the wiring that connects
...
+![](assets/blower_mounting.jpg)
+
+**STEP:3** Place the Blower Holding plate on the top of the Blower.
+
+**STEP:4** Attach the Blower Driver to the Driver mounting plate and fix the assembly to the Blower holding plate and Adapter Mounting plate by using 1" spacers for the blower wires to connect the driver.
+![](assets/blower_driver_mounting.jpg)
+
+**NOTE:** M4x80mm Lenght screws are to be useed for monting all the plates together.
+![](assets/blower_subassembly.jpg)
+
```
Comment (martukas) [inline manufacturing/blower/README.md:111]: Where are the links to all the CAD files? They should probably be in the BOM?

**Review by martukas — CHANGES_REQUESTED** [review]
There is no description for this PR.

Each of the files should be linked and explained.
The pictures are a mix of India build and US East Coast build. It should be just one and follow that sequence.

BOM of mounting components is missing.

File: .gitattributes (hunk 1 above)
Comment (RakurthiLokesh) [reply .gitattributes:14]: Solid works file which can be edited.

File: manufacturing/blower/README.md [hunk 4]
```diff
@@ -68,14 +73,26 @@ We may have parts to send you from our inventory.
 [10rs]:   https://export.rsdelivers.com/product/jst/vhr-2n/jst-vhr-female-connector-housing-396mm-pitch-2/8201172
 [11amzn]: https://www.amazon.com/gp/product/B00N51OJJ4
 [12amzn]: https://www.amazon.com/gp/product/B011JC76OA
+[13rw]:   assets/blower_adapter.stl
```
Comment (martukas) [inline manufacturing/blower/README.md:76]: but why is this one STL and the other ones DXF?

File: manufacturing/blower/README.md [hunk 5]
```diff
@@ -68,14 +73,26 @@ We may have parts to send you from our inventory.
...
+[16rw]:   assets/blower_holding_plate.dxf
+
+
 
 ## Tools
 
 | Item | Quantity | Manufacturer  | Part #              | Price (USD) | Sources         | Notes |
 | ---- |---------:| ------------- | ------------------- | -----------:|-----------------| ----- |
 | a1   |        1 | Preciva       | PR-3254 (PR102001)  |    39.99 | [Z][a1amzn]     | Crimping tool for HX and HV connectors |
+| a2   |        1 | RespiraWorks  | Blower Adapter Gasket      | $1 (material) | [B][a2rw]            | Acrylic sheet cut from data in link |
```
Comment (martukas) [inline manufacturing/blower/README.md:88]: Is it really cut from acrylic?

Comment (martukas) [reply manufacturing/blower/README.md:88]: And what material is the gasket?

Comment (martukas) [reply manufacturing/blower/README.md:88]: And are these really tools or parts?

File: manufacturing/blower/README.md [hunk 6]
```diff
@@ -68,14 +73,26 @@ We may have parts to send you from our inventory.
...
+| a3   |        1 | RespiraWorks  | Blower Gasket      | $1 (material) | [B][a3rw]            | Acrylic sheet cut from data in link |
+
 
 [a1amzn]:https://www.amazon.com/gp/product/B07R1H3Z8X
+[a2rw]:assets/blower_adapter_gasket_.dxf
+[a3rw]:assets/blower_gasket_.dxf
```
Comment (martukas) [inline manufacturing/blower/README.md:94]: And again here is the DXF, but where are the other files? What is the logic for this?

File: manufacturing/blower/README.md [hunk 7]
```diff
@@ -92,3 +109,18 @@ You will have to crimp connectors to the second end of the wiring that connects
 to the [PCB](../../pcb).
 It depends on your build layout, but in the case that you need longer wires, you will have to crimp both ends.
 In such a case you might need a greater number of XH and VH connector components listed in the BOM above.
+
+## Assembly Instructions of Blower to the Enclosure
+
+**STEP:1** Attach the Blower Driver to the Driver mounting plate and fix the assembly to the Blower holding plate and Adapter Mounting plate by using 1" spacers for the blower wires to connect the driver.
```
Comment (martukas) [inline manufacturing/blower/README.md:115]: Why not link to parts of this document where those parts are documented. And actually, why not mention parts by BOM number like on the pinch valve page?

**Review by martukas — CHANGES_REQUESTED** [review]

File: .gitattributes (hunk 1 above)
Comment (martukas) [reply .gitattributes:14]: And it should be explained in the readme. It is not just for you to answer this question to me personally this one time, but for everyone to be able to find out when they use this documentation.

**Comment by martukas** [comment]
Please do more than the bare minimum. When you submit the PR, it should look exemplary and at least as good if not better than other similar pages.

File: manufacturing/blower/README.md (hunk 5 above)
Comment (RakurthiLokesh) [reply manufacturing/blower/README.md:88]: Yeah the gaskets are cut using acrylic sheet.

**Comment by martukas** [comment]
Original author of PR is non-responsive.

**Pull request closed without merging**